src/layouts/LinkReport/components/LinkReportModal/LinkReportModal.tsx

```diff
@@ -0,0 +1,549 @@
+  const isBrokenLinksReporterEnabled = useFeatureIsOn(
+    "is_broken_links_report_enabled"
+  )
+  const {
+    data: brokenLinks,
+    error: brokenLinksError,
+    isLoading: isBrokenLinksFetching,
+  } = useGetBrokenLinks(siteName, isBrokenLinksReporterEnabled)
+
+  const isBrokenLinksLoading =
+    brokenLinks?.status === "loading" || isBrokenLinksFetching
+
+  return (
+    <Grid
+      templateColumns="repeat(12, 1fr)"
+      w="100%"
+      maxW="1440px"
+      gap="1rem"
+      mx="auto"
+      px="2rem"
+    >
+      <GridItem colSpan={1} />
+      <GridItem colSpan={10}>
+        <VStack paddingTop="3rem" width="100%" align="center" spacing=".25rem">
+          <Text
+            font-feature-settings="cv10 cv05"
+            font-variant-numeric="lining-nums tabular-nums"
+            textStyle="body-3"
+            fontFamily={typography.fontFamilies.inter}
+            overflow="hidden"
+            color={colors.base.content.default}
+            textOverflow="ellipsis"
+            alignSelf="flex-start"
+          >
+            {pageCmsUrl.split("/").at(-1)}
+          </Text>
+
+          <HStack
+            display="flex"
+            align-items="center"
+            gap="1.5rem"
+            width="100%"
+            paddingBottom="1.25rem"
+          >
+            <Text
+              fontSize="1.75rem"
+              fontStyle="normal"
+              fontWeight="600"
+              lineHeight="2.25rem"
+              fontFamily="Inter"
+              flex="1 0 0"
+              font-feature-settings="cv10 cv05"
+              alignSelf="flex-start"
+            >
+              {isBrokenLinksLoading
+                ? "Re-scanning page..."
+                : `${detailedErrorArr.length} broken references found`}
+            </Text>
+            <Link href={pageStagingUrl} isExternal>
+              <HStack spacing="1">
+                <Text>View page on staging</Text>
+              </HStack>
+            </Link>
+            <Link href={pageCmsUrl} isExternal>
+              <HStack spacing="1">
+                <Text>Edit page on CMS</Text>
+              </HStack>
+            </Link>
+          </HStack>
+          {!isBrokenLinksLoading && detailedErrorArr.length === 0 && (
+            <LinkReportModalNoBrokenLink {...props} />
+          )}
+
+          {(isBrokenLinksLoading || detailedErrorArr.length !== 0) && (
+            <>
+              <HStack
+                justifyContent="space-between"
+                display="flex"
+                width="100%"
+                paddingBottom="1.25rem"
+              >
+                <Text
+                  textStyle="body-2"
+                  fontFamily={typography.fontFamilies.inter}
+                  color="base.content.medium"
+                >
+                  After fixing the references, you can click the &quot;Run check
+                  for this page&quot; in the top right hand corner.
+                </Text>
```

Comment:
{`After fixing the references, you can click the "Run check
                  for this page" in the top right hand corner.`}